fili-wikipedia-example/src/test/resources/userConfig.properties.sample

```diff
@@ -6,6 +6,7 @@ bard__dimension_backend = memory
 
 bard__druid_cache_enabled = true
```

Comment:
Did you mean this? https://github.com/yahoo/fili/blob/ef959dab04967fd417c39fc49937654bbe47c8b7/fili-core/src/main/java/com/yahoo/bard/webservice/application/AbstractBinderFactory.java#L313